Tests/VoiceAttackPluginTests.cs

```diff
@@ -45,10 +44,10 @@ public void TestTranslateStarSystems()
         }
 
         [TestMethod]
-        [DeploymentItem(@"..\..\starsystems.txt")]
         public void TestTranslateStarSystemsStability()
         {
-            string[] starSystems = System.IO.File.ReadAllLines(@"starsystems.txt");
+            string[] starSystems = Tests.Properties.Resources.starsystems.Split(new[] { '\r', '\n' });
```

Comment:
I'd probably use a `using Tests.Properties.Resources` here. Select and Ctrl+. to make the change.